src/style.js

```diff
@@ -149,11 +150,29 @@ export function getStyle(element, property, opt_bypassCache) {
  */
 export function setStyles(element, styles) {
   for (const k in styles) {
-    setStyle(element, /*OK*/k, styles[k]);
+    setStyle(element, k, styles[k]);
   }
 }
 
 
+/**
+ * Asserts that the styles does not contain the `display` style.
+ * This is the only possible way to pass a dynamic styles object to setStyles
+ * and setImportantStyles.
```

Comment:
Let's explain *why* we don't allow `display` and add a link to a GitHub issue for more reading.